src/app/phones/page.tsx

```diff
@@ -1,13 +1,13 @@
-import grid from 'src/styles/modules/grid.module.scss'
-import titles from 'src/styles/modules/titles.module.scss';
-import paragraphs from 'src/styles/modules/paragraphs.module.scss';
+import grid from "src/styles/modules/grid.module.scss";
+import titles from "src/styles/modules/titles.module.scss";
+import paragraphs from "src/styles/modules/paragraphs.module.scss";
 import BreadCrumbs from "../../components/BreadCrumbs/BreadCrumbs";
-import phones from './page.module.scss'
+import phones from "../../styles/modules/page.module.scss";
```

Comment:
Why are all strings suddenly in double quotes? Fix this in all files in this pr, please.